plugins/env/app/assets/javascripts/env/application.js

```diff
@@ -49,14 +49,39 @@
     return $new_row;
   }
 
+  // Update the query string of the associated preview link
+  function formatPreview($preview, val) {
+    if (val) {
+      $preview[0].search = $.param({ group_id: val });
+      $preview.show();
+    } else {
+      $preview.hide();
+    }
+  }
+
+  function setupPreview($input) {
+    var $select = $input.find('select');
+    var $preview = $input.find('.checkbox a');
+
+    formatPreview($preview, $select.val());
+
+    $select.on('change', function(e) {
+      e.preventDefault();
+      formatPreview($preview, $select.val());
+    });
```

Comment:
might be able to remove the duplication by appending a `.trigger('change')` here so it always gets executed once